src/main/java/ca/serum390/godzilla/api/routers/ApiRouter.java

```diff
@@ -39,23 +38,22 @@
         return RouterFunctions.route()
                 .path("/api", apiBuilder -> apiBuilder
                     .GET("/healthcheck", req -> ok().bodyValue(ALL_GOOD_IN_THE_HOOD))
-                    .GET("/sales", salesHandler::demoSales)
+                   // .GET("/sales", salesHandler::demoSales)
```

Comment:
Remove the sales handler from the parameter list of this function and remove the uncommented line